scijava/scijava-ops-api/src/main/java/org/scijava/ops/api/OpBuilder.java

```diff
@@ -308,18 +361,32 @@ public Arity0_OV(final O out) {
 			this.out = out;
 		}
 
+		/**
+		 * @return An instance of the matched {@link org.scijava.function.Computers} op, e.g. for reuse.
```

Comment:
This description implies that a match has already taken place. In actuality, every call to this method will do another match action. I would change the javadoc accordingly to say "Match and return a computer op" or some such.

Reply:
I think it's important to have the `@return` on the terminal builder steps, when applicable. I agree with the correctness of your statement, but was trying to avoid repetition in description and return. I'm thinking maybe something like this will work?
```
/**
 * Query an Op based on the choices made with this builder.
 *
 * @return An instance of the requested Op, if matched.
 */
```

Reply:
And I agree about the desirability of having `@return` declared. I didn't mean to suggest to get rid of it, technically. We should find a way to keep it while minimizing redundancy as you say.